src/OrchardCore/OrchardCore.Abstractions/Shell/ShellSettings.cs

```diff
@@ -74,5 +75,8 @@ public TenantState State
         }
 
         public Task EnsureConfigurationAsync() => _configuration.EnsureConfigurationAsync();
+
+        public bool IsDefaultShell()
+            => String.Equals(Name, ShellHelper.DefaultShellName, StringComparison.OrdinalIgnoreCase);
```

Comment:
Later please create a subsequent PR where this is used instead of comparing to `DefaultShellName` where applicable.

Reply:
Sure